src/commands/force/user/permset/assign.ts

```diff
@@ -78,6 +78,9 @@ export class UserPermsetAssignCommand extends SfdxCommand {
         }
       }
     } catch (e) {
+      if (!(e instanceof SfdxError)) {
```

Comment:
this doesn't look right.  If it's not an SfdxError then throw it, otherwise wrap the SfdxError in another SfdxError and throw?

Reply:
I meant to check for `Error` not sfdxError.  I fixed it a different way now.